js/src/personal/bryan/Frame/NavBar.tsx

```diff
@@ -1,6 +1,6 @@
 import { Container, Group, Anchor } from '@mantine/core'
-import { Settings } from './Settings.tsx'
-import classes from './Bryan.module.css'
+import { Settings } from '../Settings/Settings.tsx'
+import classes from '@/.../personal/bryan/Bryan.module.css'
```

Comment:
MB I meant this as an example, not sure if the `@/.../` thing works?
'@/personal/bryan/Bryan.module.css' should work though

Reply:
oh yeah, tested and it didn't work 💀 
Hard resetting to previous commit XD